config/default/manager_default_images.yaml

```diff
@@ -0,0 +1,19 @@
+# This patch inject custom ENV settings to the manager container
+# Used to set our default image locations
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        env:
+        - name: WATCHER_API_IMAGE_URL_DEFAULT
+          value: quay.io/tripleoantelopecentos9/openstack-watcher-api:current-tripleo
```

Comment:
Should be pulled from quay.io/podified-antelope-centos9 with current-podified tag:

quay.io/podified-antelope-centos9/openstack-watcher-api:current-podified

Reply:
Done!